requirements.txt

```diff
@@ -2,4 +2,4 @@ numpy
 scipy>=0.9
 # scikit-learn
 pandas
-
+pybedtools
```

Comment:
I've pulled this branch onto my local and am able to get the same error. Bug replicated.